.github/workflows/ci.yml

```
@@ -226,3 +226,19 @@ jobs:
         command: check
         log-level: error
         arguments: --all-features --target ${{ matrix.platform.target }}
+
+  swc:
+    name: SWC
```

Maybe (as an outsider, I had to look up what SWC is).
```suggestion
    name: Check JavaScript
```

---

Good point!